### DISCUSSION TITLE
Referenceerror

### DISCUSSION THREAD

**Discussion by Sathishvel** [chartjs/chartjs-plugin-zoom#867] [Q&A]
In Nextjs application while I'm run the command for build and export there was an ReferenceError

I've attached the screenshot for your reference..

When remove the import f chartjs-plugin-zoom from the project, the build and export works properly..

**Comment by Sathishvel** [comment]
![IMG_20231019_194548](https://github.com/chartjs/chartjs-plugin-zoom/assets/24644668/d9e0c0e5-d27e-4fc4-9263-1d738674afd9)

**Comment by kurkle** [comment]
Try using ng-hammerjs instead of hammerjs.
